includes/DataDumpHooks.php

```diff
@@ -25,4 +25,14 @@ public static function onLoadExtensionSchemaUpdates( DatabaseUpdater $updater )
 			__DIR__ . '/../sql/patches/patch-dumps_size.sql'
 		);
 	}
+
+	public static function onNewSidebarItem( $skin, &$bar ) {
+		if ( isset( $bar['Administration'] ) && $bar['Administration'] ) {
```

Comment:
Why check if an array exists to add to it, when you can add to it and create it?